docs/python-sdk/changelog.md

```diff
@@ -7,6 +7,27 @@ sidebar_position: 8
 
 # Changelog
 
+## v1.17.0 (2025-04-10)
+
+**[`fused-py`](/python-sdk/)**
+
+- Team UDFs can be loaded or run by specifying the name "team", as in: `fused.load("team/udf_name")`
+- `Udf.to_fused` supports overwriting the UDF when saving.
+- Added `fused.enable_gcs()` to configure using the Google Cloud Platform secret specified in Fused secret manager.
+- `@fused.cache` locking mechanism has changed and will not allow multiple concurrent runs.
+- Upgraded DuckDB to v1.2.2.
+
```

Comment:
Some more potential things to mention:

```suggestion
- Running a saved UDF by token or name will now also show the logs (print statements and error tracebacks).
- All functions interacting with the Fused server will now retry automatically (by default 3 times)
- Python 3.9 support is deprecated. The next release of `fused` will require Python 3.10+
```

and should we mention the removal of `fused_batch`?